pretix_mollie/payment.py

```diff
@@ -836,19 +961,30 @@ def execute_refund(self, refund: OrderRefund, retry=True):
             body["testmode"] = refund.payment.info_data.get("mode", "live") == "test"
         try:
             refresh_mollie_token(self.event, True)
-            req = requests.post(
-                "https://api.mollie.com/v2/payments/{}/refunds".format(payment),
-                json=body,
-                headers=self.request_headers,
-            )
+            if refund.payment.info_data.get("resource") == "order":
+                body = {
+                    "lines": [],
+                    # "If you send an empty array, the entire order will be refunded."
+                }
+                req = requests.post(
+                    "https://api.mollie.com/v2/orders/{}/refunds".format(order),
```

Comment:
`order` is undefined - it's probably still `payment` here (which however sounds ambiguous since it can either be the Mollie payment ID or the Mollie order ID. Perhaps we just meet in the middle and rename it to something like `resource_id` and use it for both payment and order?

Reply:
m) thanks for noticing